packages/account/src/Components/additional-kyc-info-modal/additional-kyc-info-form.tsx

```diff
@@ -0,0 +1,248 @@
+import React from 'react';
+import {
+    Autocomplete,
+    Button,
+    DesktopWrapper,
+    Loading,
+    MobileWrapper,
+    Modal,
+    SelectNative,
+    Text,
+} from '@deriv/components';
+import { observer, useStore } from '@deriv/stores';
+import { Localize } from '@deriv/translations';
+import classNames from 'classnames';
+import { isMobile } from '@deriv/shared';
+import { Field, FieldProps, Form, Formik, FormikErrors } from 'formik';
+import FormFieldInfo from '../form-field-info';
+import { FormInputField } from '../forms/form-fields';
+import { TListItem, getFormConfig } from './form-config';
+import { useSettings } from '../../../../api/src/hooks';
+
+const FormTitle = () => (
+    <Text
+        as='p'
+        size='s'
+        line_height='xxl'
+        align={isMobile() ? 'left' : 'center'}
+        className='additional-kyc-info-modal__form--header'
+    >
+        <Localize i18n_default_text='Please take a moment to update your information now.' />
+    </Text>
+);
+
+type TAdditionalKycInfoFormProps = {
+    setError?: React.Dispatch<React.SetStateAction<string>>;
+};
+
+export const AdditionalKycInfoForm = observer(({ setError }: TAdditionalKycInfoFormProps) => {
+    const { client, ui, notifications } = useStore();
```

Comment:
you might want to take out `refreshNotifications` from `notifications` here; instead of using dot notation below.
we'll follow the current standard way of destructuring and using store.
`const {notifications: notifications_array, refreshNotifications }  = notifications;`

Reply:
It's just personal preference as it doesn't hurt to write this way or please correct me if there's a set rule to write like this

Reply:
i saw similar comment on other PRs. so thought it would be better if similar structure is followed in all files in the project😄 
most of my review comments will be based on what i saw or got from seniors 📝